src/pages/admin/locations/create.tsx

```diff
@@ -1,54 +1,17 @@
-import apiClient from "@api/client";
-import { CreateLocationRequest } from "@api/client/models/CreateLocationRequest";
-import { Location } from "@api/client/models/Location";
-import ROUTES from "@common/routes";
-import FormWrapper from "@components/FormWrapper";
-import LocationEditor from "@components/LocationEditor";
-import Page from "@components/Page";
+import LocationDetailsPage from "@components/LocationDetailsPage";
 import { loadMessages } from "@utils/i18n";
 import withAuth from "@utils/withAuth";
-import { GetStaticProps } from "next";
-import { useRouter } from "next/router";
-import { FormEvent, useState } from "react";
-import { useTranslations } from "use-intl";
+import { GetServerSideProps } from "next";
 
-export const getStaticProps: GetStaticProps = async (context) => ({
+interface Props {
+	location: null;
+}
+
+export const getServerSideProps: GetServerSideProps<Props> = async (context) => ({
```

Comment:
Hier sollten wir (analog zu `src/pages/admin/attractions/create.tsx`) auch die Wrapper-Funktion `withApiClientAndPagination` verwenden. Die sorgt auch dafür, dass der Access Token validiert und ggf. zum Login umgeleitet wird! (Denn diese Route soll ja auch nur für eingeloggte Benutzer:innen erreichbar sein.)

Reply:
**Update:** Habe gerade gemerkt, dass sich das Format der `withApiClientAndPagination`-Funktion nochmal ein bisschen in https://github.com/technologiestiftung/kulturdaten-webapp/pull/29 ändert! Nur als Hinweis. ✌️